assigner/commands/get.py

```diff
@@ -70,17 +70,23 @@ def get(conf, args):
                         result.note
                     )
 
-                    # see http://gitpython.readthedocs.io/en/stable/reference.html#git.remote.FetchInfo
+                    # see:
+                    # http://gitpython.readthedocs.io/en/stable/reference.html#git.remote.FetchInfo
                     if result.flags & result.NEW_HEAD:
-                        output.add_row([row, sec, sid, name, "{}: new branch at {}".format(
-                            result.ref.name, str(result.ref.commit)[:8]
-                        )])
+                        output.add_row([
+                            row, sec, sid, name, "{}: new branch at {}".format(
+                                result.ref.name, str(result.ref.commit)[:8]
+                            )
+                        ])
                         row = sec = sid = name = "" # don't print user info more than once
 
                     elif result.old_commit is not None:
-                        output.add_row([row, sec, sid, name, "{}: {} -> {}".format(
-                            result.ref.name, str(result.old_commit)[:8], str(result.ref.commit)[:8]
-                        )])
+                        output.add_row([
+                            row, sec, sid, name, "{}: {} -> {}".format(
```

Comment:
Every time I see this line, the `{}: {} -> {}` bit fills me with hope that I've somehow stumbled into a haskell codebase

Reply:
Yeah, it looks like a js lambda for me :D